ghlicense/cmd.py

```diff
@@ -333,6 +337,13 @@ def main():
             "Repos without License and forked: " + str(count_forked) + "\n")
         report_file.write("Total Repos: " +
                           str(count_no_license + count_license) + "\n")
+        
+        if ARGS.html is not None: 
+            with open("report.html", "w") as e:
+                for lines in report_file.readlines():
```

Comment:
Not working since your reading a file that is not closed yet (tested on python 3.10, may work elsewhere)
You'll have to close the `report_file` before, then reopen it in read mode and use `report_file.read()` simply